Jenkinsfile

```diff
@@ -215,8 +230,16 @@ pipeline {
             Branch: ${BRANCH_NAME}
           """
           if (isTriggered()) {
-            buildInfo += """
+            if (hasUpstream()) {
+              buildInfo += """
             Triggered by: ${upstreamJobName} #${upstreamBuildNumber}
+            """
+            } else {
+              buildInfo += """
+            Originally triggered by a ${NUXEO_LTS_JOB} upstream build.
+            """
```

Comment:
```suggestion
              buildInfo += """
              Triggered by: ${upstreamJobName} #${upstreamBuildNumber}
              """
            } else {
              buildInfo += """
              Originally triggered by a ${NUXEO_LTS_JOB} upstream build.
              """
```

Reply:
In fact, I think it is OK as we want to have something aligned in the build console.